CONTRIBUTING.md

```diff
@@ -27,33 +35,38 @@ cd liveask
 ```
 
 ## Running The Local Instance
-### Dependencies
-**First Terminal**
+
+> [!IMPORTANT]
+> You need to open three terminal tabs/instances and run the following commands in each of them.
```

Comment:
Thats not fully accurate anymore. you can use the roots justfile and *just* run `just run` ^^
That will spin up the server and the frontend using `parallelrun`

Reply:
I would suggest we focus on that way of running liveask these days locally. so its boiled down to running in two terminals

Reply:
@extrawurst I have made the changes so that the contributor knows they need to install `parallelrun` and run `just run` in the second terminal for both the backend and frontend.